recipe/meta.yaml

```diff
@@ -56,11 +56,13 @@ requirements:
     - pyqt  # [not osx]
     - tk 8.5.*  # [linux]
     - functools32  # [py2k]
+    - setuptools  # [py35]
```

Comment:
@ericdill do you know if only matplotlib on Python 3.5 uses `pkg_resources` or the issue does not manifest itself b/c for some reason only `conda-forge`'s does not have `setuptools`? If so we should remove the `[py35]` selector.